eloquent-mutators.md

```diff
@@ -267,7 +267,9 @@ When defining a `date` or `datetime` cast, you may also specify the date's forma
         'created_at' => 'datetime:Y-m-d',
     ];
 
-When a column is cast as a date, you may set its value to a UNIX timestamp, date string (`Y-m-d`), date-time string, or a `DateTime` / `Carbon` instance. The date's value will be correctly converted and stored in your database:
+> {note} It's important to note that when using a custom date format using the `date` or `datetime` cast that the time will not be converted to UTC when the date is serialized.
```

Comment:
Nope, it's the contrary:
- With a custom format: app.timezone is applied
- Without a custom format: UTC is enforced as per ISO-8601 norm (as it's the format used by default for JSON and maximizing interoperability)

Reply:
@kylekatarnls I don't think that's true. Taylor and I both verified that `datetime` cast with a custom format ignores the UTC conversion. That's the real explanation.

Reply:
Sorry, but this technical explanation is helpless for the reader. It does not tell anything about the concrete results and difference you'll get.

Reply:
Some first other topic to document would be how Laravel creates date objects from DB datetime strings and what timezone it picks to read those strings (e.g. `config('app.timezone')`).

Then for the custom date format, I think both scenario should be mentioned (with sentence that does not use negation), so the reader is not left into the doubt about *what happen in the other cases*:
```suggestion
> {note} It's important to note that `date` or `datetime` properties are output by default as UTC ISO-8601 string, then as soon as a custom date format is used, UTC will no longer be enforced, so inner timezone of the `Carbon` instance applies (refer to [How DB datetime fields are converted to Carbon instances section])
```